docs/usage/CLEANUP_GUIDE.md

```diff
@@ -0,0 +1,24 @@
+# Cleanup of old unresolved reference custom objects
+
+Keeping the old custom objects around forever can negatively influence the performance of your project and the time it takes to restore it from a backup.  
+Deleting unused data ensures the best performance for your project. 
+
+The cleanup of unresolved references custom objects queries the custom objects with `container="commercetools-sync-java.UnresolvedReferencesService.productDrafts" and container="commercetools-sync-java.UnresolvedTransitionsService.stateDrafts""` and their `lastModifiedAt` values. For example, you could configure the `deleteDaysAfterLastModification` parameter to 30 days to delete the custom objects if they haven't been modified in the last 30 days.
```

Comment:
More than a review, It's a question. Here how can we know if we haven't modified since last 30 days? Or is there any way that we can get to know from DB that since how many days we haven't modified the custom objects?
I am asking this question because you have randomly used 30 days.

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/651/commits/ccac39acc2652b565e6ce8da4d73e29f0b4ad0d0